guide/src/building_and_distribution.md

```diff
@@ -152,10 +152,14 @@ As your extension module may be run with multiple different Python versions you
 
 PyO3 is only able to link your extension module to api3 version up to and including your host Python version. E.g., if you set `abi3-py38` and try to compile the crate with a host of Python 3.7, the build will fail.
 
-As an advanced feature, you can build PyO3 wheel without calling Python interpreter with the environment variable `PYO3_NO_PYTHON` set. On unix systems this works unconditionally; on Windows you must also set the `RUSTFLAGS` evironment variable to contain `-L native=/path/to/python/libs` so that the linker can find `python3.lib`.
-
 > Note: If you set more that one of these api version feature flags the lowest version always wins. For example, with both `abi3-py37` and `abi3-py38` set, PyO3 would build a wheel which supports Python 3.7 and up.
 
+As an advanced feature, you can build PyO3 wheel without calling Python interpreter with the environment variable `PYO3_NO_PYTHON` set.
+On Unix-like systems this works unconditionally; on Windows you must also set the `RUSTFLAGS` evironment variable to contain
+`-L native=/path/to/python/libs` so that the linker can find `python3.lib`. Alternatively, an experimental `generate-abi3-import-lib`
+crate feature may be enabled, and the DLL import library will be created automatically by PyO3.
```

Comment:
Do I still need to modify `RUSTFLAGS` when the feature is enabled?

Reply:
And if so, could we amend the build script to automatically set the search when we import library is automatically genereted?

Reply:
> Do I still need to modify `RUSTFLAGS` when the feature is enabled?

No, when the feature is enabled nothing else is required. I'll try to reword the sentence it to make that clearer.

Reply:
I've rewritten this part as a separate paragraph.